provisioning_templates/snippet/redhat_register.erb

```diff
@@ -3,6 +3,8 @@ kind: snippet
 name: redhat_register
 model: ProvisioningTemplate
 snippet: true
+-%>
+<%#
 # Red Hat Registration Snippet
 #
 # General parameters:
```

Comment:
Is the indentation off or is it just the diff-view? :eyeglasses:

Reply:
it starts on col 0 in my editor